ngraph/src/ngraph/frontend/onnx_import/op/round.cpp

```diff
@@ -30,10 +30,36 @@ namespace ngraph
         {
             namespace set_1
             {
+                // WARNING!
+                // Current version is:
+                // data_floor = floor(data)
+                // diff = data - data_floor
+                // if(diff < 0.5f)
+                //   return data_floor
+                // else
+                //   return data_floor + 1.0f
+                //
+                // The correct version should contain condition:
+                // if (diff < 0.5f || (diff == 0.5f && static_cast<int>(data_floor) % 2 == 0))
```

Comment:
I just wonder, why we can't implement this additional condition with the Mod operator?

Reply:
@mitruska I've tried to do it, but it fails during decomposition of `Mod` operator (`Mod ` is fused operator) - because of not known shape during decomposition.
What's more the produced subgraph was pretty complicated for rather simple operation.
But I can check it once more (maybe it is possible to extend shape propagation for this case)

Reply:
Sure, I understand